automation_tools/__init__.py

```diff
@@ -1797,11 +1792,9 @@ def apply_hotfix():
         run('/root/hotfix.sh')
 
 
-def upstream_install(admin_password=None, run_katello_installer=True):
-    """Task to install Foreman nightly using forklift scripts"""
-    koji = 'koji' in os.environ.get('DISTRIBUTION', '').lower()
-    if admin_password is None:
-        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+def upstream_install(admin_password='changeme', run_katello_installer=True, distribution=None):
+    """Task to install Nightly using forklift playbook"""
```

Comment:
Would it make sense to add description of the parameters into the docstring? Maybe at last the new one?

Reply:
Ok, nice tip:-)

Reply:
Added for ages omitted params ```admin_password``` and ```run_katello_installer``` into docstrings